tests/pipelines/test_pipelines.py

```diff
@@ -327,28 +327,28 @@ def test_download_no_openvino_by_default(self):
     def test_download_no_onnx_by_default(self):
         with tempfile.TemporaryDirectory() as tmpdirname:
             tmpdirname = DiffusionPipeline.download(
-                "hf-internal-testing/tiny-random-OnnxStableDiffusionPipeline",
+                "hf-internal-testing/tiny-stable-diffusion-pipe-indexes",
```

Comment:
I don't think there are any `onnx` files present in this. So this test isn't verifying that the onnx files are being ignored by default. You can try using 
this repo `hf-internal-testing/tiny-stable-diffusion-xl-pipe`